### PR TITLE
[Outlook](SSO): Remove preview info

### PR DESCRIPTION
**Pull request by davidchesnut**
Removes some old info indicating SSO is preview (since it is actually GA now.)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d928e45](https://github.com/OfficeDev/office-js-docs-pr/commits/d928e45c8215a6f6a1d342acc101146b1b2f1717)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-fix-preview-comments/docs/outlook/authentication.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/authentication?branch=pr-en-us-2308) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011062215096696-2308%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CoEEshavzWQrS381h995sMC7PvQCK2ABzbK%2BTi5Abc0%3D&st=2020-11-06T22%3A10%3A41Z&se=2020-12-07T22%3A15%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/authentication.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ Callback tokens provide access to the user's mailbox from your server back-end,
 
 - Needs access to the user's mailbox from your server back-end.
 
-Add-ins obtain callback tokens using one of the [getCallbackTokenAsync](../reference/objectmodel/preview-requirement-set/office.context.mailbox.md#methods) methods. The level of access is controlled by the permissions specified in the add-in manifest.
+Add-ins obtain callback tokens using one of the [getCallbackTokenAsync](https://docs.microsoft.com/javascript/api/outlook/office.mailbox#getcallbacktokenasync-options--callback-) methods. The level of access is controlled by the permissions specified in the add-in manifest.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/authentication.md:64]:
```suggestion
Add-ins obtain callback tokens using one of the [getCallbackTokenAsync](../reference/objectmodel/requirement-set-1.9/office.context.mailbox.md#methods) methods. The level of access is controlled by the permissions specified in the add-in manifest.
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
Thanks for making this update

**Comment by davidchesnut** [comment]
already made this same change in pr #2319

**Pull request closed without merging**